src/ring_theory/fintype.lean

```diff
@@ -0,0 +1,17 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import algebra.associated
+import data.fintype.basic
+
+/-!
+Some facts about finite rings.
```

Comment:
```suggestion
# Some facts about finite rings.
```

Reply:
(I edited the suggestion to remove a period.)